src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$primary-text-color: #060B35;
+$secondary-text-color: #616070;
+$blue-color: #00ACDC;
```

Comment:
Variable names should describe their purpose. As if the designer changes the color to red instead, it will be strange when `$color-blue` will be red. For example `$main-color`